django_db_schema/hospital/models.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models
+
+
+class Hospital(models.Model):
+    name = models.CharField(max_length=100, null=False, blank=False)
+    city = models.CharField(max_length=100, null=False, blank=False)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        indexes = [
+            models.Index(fields=['name'])
```

Comment:
I prefer using `db_index=True`
for example
`name = models.CharField(db_index=True, ...)`

Reply:
d